mne/viz/eyetracking/tests/test_heatmap.py

```diff
@@ -0,0 +1,42 @@
+# Authors: Scott Huberty <seh33@uw.edu>
+#
+# License: Simplified BSD
+
+import pytest
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import mne
+
+
+@pytest.mark.parametrize(
+    "axes",
+    [(None), (True)],
+)
+def test_plot_heatmap(axes):
+    """Test plot_gaze."""
+    # Create a toy epochs instance
+    info = info = mne.create_info(
+        ch_names=["xpos", "ypos"], sfreq=100, ch_types="eyegaze"
+    )
+    # here we pretend that the subject was looking at the center of the screen
+    # we limit the gaze data between 860-1060px horizontally and 440-640px vertically
+    data = np.vstack([np.full((1, 100), 1920 / 2), np.full((1, 100), 1080 / 2)])
```

Comment:
```suggestion
    # simulate a steady fixation at the center of the screen
    width, height = (1920, 1080)
    shape = (1, 100)  # x or y, time
    data = np.vstack([np.full(shape, width / 2), np.full(shape, height / 2)])
```

comment was misleading. We're not limiting the data between anything: we're setting a constant value.